doc/book/replication/repl_leader_elect.rst

```diff
@@ -83,11 +83,6 @@ between each node pair so as it would be the full mesh topology. This is needed
 because election messages for voting and other internal things need direct
 connection between the nodes.
 
-Also, if election is enabled on the node, it won't replicate from any nodes except
-the newest leader. This is done to avoid the issue when a new leader is elected,
-but the old leader has somehow survived and tries to send more changes
-to the other nodes.
-
```

Comment:
Is this deletion intended? And the other one in this file too.
I'm asking because it's not related to the issue.

Reply:
Oops, my mistake. Will revert this change.